### DISCUSSION TITLE
Road map?

### DISCUSSION THREAD

**Discussion by Horkas** [tsunamayo/Starship-EVO#4471] [General]
Is there anywhere where we can find a planned roadmap for the game?

I know things change all the time and priority's shift. Sometimes planned things just don't make the cut, so I get why allot of developers don't like posting them. But they are absolutely invaluable to players, helping them keep track of the games progress and give feedback that isn't already planned.

Sometimes, planned changes/additions might be making the game worse off than intended, (Starmade is the perfect example of this.) Feedback on planned changes is very important to making a fun game!

**Comment by Briaireous** [comment]
nope

**Comment by Blodgemeister** [comment]
This is a shame. While a strict time-line may not suite Tsuna's work-style (which is fine, whatever works for him) an idea of what he is at least working on next, or intends to add over a period of say 6 months would satisfy a lot of forum questions.
